### ISSUE TITLE
Upgrade: Create Question banks in courses

### ISSUE THREAD

**Issue by mattporritt** [catalyst/moodle-MDL-70329#91]
Once Issue #90 is implemented. We then need to create a mod_qbank instance in each course. 

We will need to migrate the questions into these new question banks, but this will be a separate issue.

We need to:
* [x] Create a mod_qbank instance for every course created in issue #90 
* [x] Create a mod_qbank instance for every course that has an existing course level question bank

**Comment by guillogo** [comment]
Hi @mattporritt and @safatshahin PR for issue 91 is ready: https://github.com/catalyst/moodle-MDL-70329/pull/115 however, now is adev branch as this PR depends on Issue: 90: https://github.com/catalyst/moodle-MDL-70329/pull/113

This PR will create a mod_qbank for each course created in issue 90 and also 1 for the system course and 1 for course and module context.

Thank you.

**Comment by mattporritt** [comment]
@guillogo PR #115 approved and merged

**Comment by guillogo** [comment]
Hi @mattporritt this is good and can be closed now.
